### ISSUE TITLE
Error when wrapping a redux connected component

### ISSUE THREAD

**Issue by solomon23** [mjancarik/shallow-with-context#4]
I'm getting the error
`TypeError: The defined Component must be function. You give type of object.`

When trying to use it with a HOC connected redux component

`export default connect(mapStateToProps, mapDispatchToProps)(QueryGridProvider)`

`withContext(QueryGridProvider, context)`

**Comment by mjancarik** [comment]
Hi, 

Thanks for report. Could you send me all your test case snippet or example for creating better solution.

**Comment by mjancarik** [comment]
Hi,

I try it but without your snippet I don't know why do you need shallow rendering. Because I have got something like this.

``` jsx
<ContextProvider value={{...}}>
      <ClassComponent store={{...}} todos={{...}} addTodo={[Function]} />
</ContextProvider>
```

What do you want to assert in your test?

**Comment by mjancarik** [comment]
It's should be fixed in shallow-with-context@0.2.2.
